### ISSUE TITLE
[BUG] `pass` statement in `finally` removed causing syntax errors

### ISSUE THREAD

**Issue by radoering** [hadialqattan/pycln#142]
**Describe the bug** A clear and concise description of what the bug is.

Under certain circumstances, `pycln` removes `pass` statements, which can lead to syntax errors in code.
Seems similar to #99, however for `try`-`except`-`finally`.

**To Reproduce** Steps to reproduce the behavior:

1. Take this code snippet:

   ```python
   import os

   def f():
       try:
           x = 1
           return
       finally:
           pass
   ```

2. Run this Pycln command:
   ```bash
   $ pycln test_file.py
   ```
3. Error traceback or unexpected output (if present):

   ```bash

   ```

4. Unexpected fixed code (if present):

   >

   ```python
   def f():
       try:
           x = 1
           return
       finally:

   ```

**Expected behavior**:

1.  Description: A clear and concise description of what you expected to happen.

     no syntax errors

3.  Expected output (if present):

    ```bash

    ```

4.  Expected fixed code (if present):

    >

    ```python
    def f():
        try:
            x = 1
            return
        finally:
            pass

    ```

**Environment (please complete the following informations):**

- Python Version: Python 3.9
- Pycln Version: v1.3.3
- OS Type: Windows

**Additional context** Add any other context about the problem here.

**Comment by hadialqattan** [comment]
Hi @radoering, I've fixed this bug via PR #143.

Thanks a lot for being around and supporting the project.

**Comment by hadialqattan** [comment]
Fixed in [v1.3.4](https://hadialqattan.github.io/pycln/#/CHANGELOG?id=_134-2022-06-22)

**Comment by radoering** [comment]
Thank you for the quick fix. 👍 🚀
